src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallControllerTest.kt

```diff
@@ -103,13 +105,76 @@ class RecallControllerTest {
+    every { tokenExtractor.getTokenFromHeader(bearerToken) } returns Token(::UserId.random().toString())
+    val userDetails = mockk<UserDetails>()
+    every { userDetailsService.get(any()) } returns userDetails
+    every { userDetailsService.find(any()) } returns userDetails
+    every { userDetails.caseworkerBand } returns CaseworkerBand.FOUR_PLUS
+    every { userDetails.fullName() } returns "Mickey Mouse"
+
+    val results = underTest.findAll(bearerToken)
+
+    assertThat(results.size, equalTo(7))
 
-    val results = underTest.findAll()
+    assertThat(
+      results,
+      List<RecallResponse>::containsAll,
+      listOf(
+        RecallResponse(beingBookedOnRecall.recallId(), beingBookedOnRecall.nomsNumber, createdByUserId, now, now, FirstName("Barrie"), null, LastName("Badger"), NameFormatCategory.FIRST_LAST, Status.BEING_BOOKED_ON),
+        RecallResponse(bookedOnRecall.recallId(), bookedOnRecall.nomsNumber, createdByUserId, now, now, FirstName("Barrie"), MiddleNames("Barnie"), LastName("Badger"), NameFormatCategory.FIRST_MIDDLE_LAST, Status.BOOKED_ON, bookedByUserId = bookedByUserId),
+        RecallResponse(inAssessmentRecall.recallId(), inAssessmentRecall.nomsNumber, createdByUserId, now, now, FirstName("Barrie"), MiddleNames("Barnie"), LastName("Badger"), NameFormatCategory.FIRST_MIDDLE_LAST, Status.IN_ASSESSMENT, bookedByUserId = bookedByUserId, assignee = assignee, assigneeUserName = "Mickey Mouse"),
+        RecallResponse(stoppedRecall.recallId(), stoppedRecall.nomsNumber, createdByUserId, now, now, FirstName("Barrie"), MiddleNames("Barnie"), LastName("Badger"), NameFormatCategory.FIRST_MIDDLE_LAST, Status.STOPPED, bookedByUserId = bookedByUserId, agreeWithRecall = AgreeWithRecall.NO_STOP),
+        RecallResponse(recallNotificationIssuedRecall.recallId(), recallNotificationIssuedRecall.nomsNumber, createdByUserId, now, now, FirstName("Barrie"), MiddleNames("Barnie"), LastName("Badger"), NameFormatCategory.FIRST_MIDDLE_LAST, Status.RECALL_NOTIFICATION_ISSUED, recallNotificationEmailSentDateTime = now),
+        RecallResponse(dossierInProgressRecall.recallId(), dossierInProgressRecall.nomsNumber, createdByUserId, now, now, FirstName("Barrie"), MiddleNames("Barnie"), LastName("Badger"), NameFormatCategory.FIRST_MIDDLE_LAST, Status.DOSSIER_IN_PROGRESS, recallNotificationEmailSentDateTime = now, assignee = assignee, assigneeUserName = "Mickey Mouse"),
+        RecallResponse(dossierIssuedRecall.recallId(), dossierIssuedRecall.nomsNumber, createdByUserId, now, now, FirstName("Barrie"), MiddleNames("Barnie"), LastName("Badger"), NameFormatCategory.FIRST_MIDDLE_LAST, Status.DOSSIER_ISSUED, dossierCreatedByUserId = dossierCreatedByUserId)
+      )
+
+    )
+  }
 
-    assertThat(results, equalTo(listOf(recallResponse)))
+  @Test
+  fun `gets all recalls for a band THREE returns only appropriate recalls`() {
```

Comment:
Should the test name identify what the 'appropriate' recalls are?  That would help to document the feature.